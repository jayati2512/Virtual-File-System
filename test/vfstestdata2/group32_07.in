createvfs vfs 
mountvfs vfs
createvfs iiitb 1024
mountvfs iiitb
makedir / /
makedir /a/b/c/d/e/  f
deletedir /a/b/c/d/f/h
movedir /a/ /c/ 
listdir /a/ 1 /tmp/out.txt  
unmountvfs vfs 
unmountvfs iiitb
