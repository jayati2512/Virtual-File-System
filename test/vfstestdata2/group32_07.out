createvfs_FAILURE VFS_INSUFFICIENT_ARGUMENTS
mountvfs_FAILURE DATA_FILE_NOT_FOUND
createvfs_SUCCESS
mountvfs_SUCCESS
makedir_FAILURE INVALID_CHARACTER_IN_DIRNAME
makedir_SUCCESS
deletedir_FAILURE CANNOT_FIND_SPECIFIED_DIR
movedir_FAILURE CANNOT_FIND_SPECIFIED_DESTINATIONDIR
listdir_SUCCESS
unmountvfs_FAILURE DATA_FILE_NOT_FOUND
unmountvfs_SUCCESS
